eng/pipelines/maintenance-packages.yml

```diff
@@ -40,6 +40,10 @@ extends:
       binskim:
         enabled: true
         scanOutputDirectoryOnly: true
+      policheck:
+        enabled: true
+      tsa:
+        enabled: true
```

Comment:
Do these trigger failures in public PRs or is it happening in the internal rolling builds?

Reply:
No, only official builds.